chucks-pick3-web/angular-app/src/app/components/sign-up.component/sign-up.component.ts

```diff
@@ -1,19 +1,47 @@
 import {Component, OnInit} from '@angular/core';
-
+import {FormValidationService} from '../../services/form-validation.service/form-validation.service';
+import {MemberService} from '../../services/member.service/member.service';
```

Comment:
Lets ensure all path start with `src/app/`; so the path should be like so: `'src/app/services/member.service/member.service'`.  Please update all other paths are corrected in all TS files.

Reply:
Added src/app/ to paths.